<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Dr. Tiziana A. Gelmi Candusso</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="current_research.html">Current Research</a>
</li>
<li>
  <a href="past_research.html">Past Research</a>
</li>
<li>
  <a href="software.html">Programming</a>
</li>
<li>
  <a href="public_outreach.html">Public Outreach</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/tgelmi-candusso">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/t_gelmicandusso">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/tagcandusso/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<p>On April 2019, I was invited by Prof. Kim McConkey to create a series of posts on plant spatial genetics for the Frugivore and Seed dispersal (FSD) science communication page. Here is the thread of posts</p>
<hr />
<p>April 1, 2019</p>
<p>Hi everybody, I’m Tiziana, and this month, I’m taking over this month to talk about frugivores and plant genetics.</p>
<p>I did my PhD on frugivore behavior and plant spatial genetics. I focused on tamarins of the Amazon Rainforest in Peru, where I grew up. It was nice to go back to my home country, to figure out that the relationship between frugivore behavior and plant spatial genetics was much more complicated than I thought.</p>
<p>Animal behavior and plant characteristics continually influence each other on a causal loop, making the outcome in terms of plant spatial genetics very case specific.</p>
<p>Join me this month to look into specific cases where the effect of animal behavior on plant spatial genetics has been analyzed, and hopefully by the end of the month we will end up with a general understanding on the different ways in which animals determine plant spatial genetics, what this means and why this is important.</p>
<p><a href="images/monkey1%20(1).jpg"><img src="images/monkey1%20(1).jpg" style="width:30%; margin-right: 20px"></a> <a href="images/monkey1%20(2).jpg"><img src="images/monkey1%20(2).jpg" style="width:30%; margin-right: 20px">&gt;</a></p>
<hr />
<p>April 3, 2019</p>
<p>Gene flow in plants happens through pollination and seed dispersal. Pollination will determine the intensity of genetic relatedness between individuals of the next generation, and seed dispersal will determine the spatial distribution of these individuals. Of course, in the long run the distribution of individuals influences the probability of pollen flow between individuals, therefore seed dispersal, in the long run, also influences the intensity in genetic relatedness between individuals.</p>
<p>When we want to understand the effects of seed dispersal on plant genetics, we should look into plant spatial genetics. i.e. How are plants genetically related to each other in relationship to their position.</p>
<p>The study of plant spatial genetics is comprised within the subject of landscape genetics, which assesses gene flow at the landscape level. Several approaches may be used for this:</p>
<ol style="list-style-type: lower-alpha">
<li>Landscape distance/resistance approach: correlating genetic distance (e.g. Nason’s Fij coefficient) with geographic or landscape distance to identify barriers to gene flow.</li>
<li>Overlay approach: clustering individuals into genetic groups (e.g. Bayesian cluster analysis), detecting genetic discontinuities and overlaying these onto maps describing landscape features.</li>
<li>Parentage analysis (e.g. with CERVUS) or assignment tests (e.g. GENECLASS) to determine any current relationships between areas sampled. (This method assesses contemporary gene flow rather than historical gene flow)</li>
</ol>
<p>In the context of frugivores and seed dispersal, we can use these approaches to overlay plant genetic information to animal behavior and animal movement data. We’ll be looking into this in the next posts.</p>
<p>For more on landscape genetics and the figure below, read <a href="https://doi.org/10.1016/j.tplants.2010.09.002">Holderegger et al, 2010</a></p>
<p><a href="images/April2.jpg"><img src="images/April2.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 4, 2019</p>
<p>The first method we described in the last post, is the one used to determine the presence of spatial genetic structure.</p>
<p>Spatial genetic structure defines the presence of a non-random distribution of genotypes throughout the population (i.e. individuals that are more genetically related are also spatially closer). The presence and degree of spatial genetic structure within a population can be related to local gene flow restriction and seed dispersal dynamics restricting such gene flow.</p>
<p>Vekemans and Hardy (2004) were the first to put into relationship plant life history traits to spatial genetic structure. They found that wind dispersed plants had a lower intensity of spatial genetic structure than animal dispersed plants. They didn’t really distinguish between animals though.</p>
<p>Hardy et al. (2006) put again into relationship seed dispersal and spatial genetic structure, in 10 plant species, and found scatter-hoarding and gravity dispersed plants had stronger spatial genetic structure than frugivore dispersed plants. They put scatter-hoarding and gravity in the same bag and all frugivores in another bag.</p>
<p>Finally, Gelmi-Candusso et al. (2017) distinguished between plants dispersed by different animal vectors, without the confounding factor of barochory (i.e. gravity) and distinguishing between different animal behavior. The results confirmed endozoochory (i.e. frugivores) was in fact related to weaker spatial genetic structure than synzoochory (e.g. scatter-hoarding animals) (Fig 1(a)).</p>
<p>Most importantly, the results showed that, even though most spatial genetic structure papers describe their plant’s dispersal vector vaguely in terms of taxonomy, disperser taxonomy had little to do with plant spatial genetic structure (Fig 1(b)).</p>
<p>Instead, the results showed that regardless of the taxonomic affiliation of the animal, spatial genetic structure was mostly related to animal foraging behavior. When the animal vector had long feeding bouts or short post-feeding movements (Fig.1(c) B) the plant had stronger spatial genetic structure than when the animal had short feeding bouts or long post-feeding/seed collecting movements (Fig 1(c) A,E).</p>
<p>However, there was still great variation within groups of animals with behavior that led to an accumulation of seeds (i.e. roosting; larder-hoarders) (Fig 1(c) C,D). A potential cause is maternal source diversity of accumulated seed has a stronger effect on adult spatial genetic structure than seed dispersal distance.</p>
<p>We’ll learn more about how specific cases of frugivore behavior that leads to the accumulation of seeds affects spatial genetic structure in the next posts.</p>
<p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14351">Gelmi-Candusso (2017) Effects of zoochory on the spatial genetic structure of plant populations. Molecular Ecology</a></p>
<p><a href="images/April4.jpg"><img src="images/April4.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 7, 2019</p>
<p>On the last post, we discussed how spatial genetic structure is highly variable, in terms of intensity, in plants whose seeds have been accumulated by animals.</p>
<p>Here we see the example of the spatial genetic structure created by white-bellied Spider Monkeys in a Palm in Barro Colorado Island, Panama</p>
<p><a href="http://dx.doi.org/10.1016/j.actao.2015.07.005">Karubian et al (2015)</a> found strong spatial genetic structure in seeds of Oenocarpus bataua deposited beneath the sleeping sites of white-bellied spider monkeys, which are used repetitively. Within these, a low number of maternal sources was genetically identified.</p>
<p>These results were contrary to their expectations given spider monkeys’ wide range of movements, their fluid social organization and the use of multiple sleeping trees over time, but were attributable to their daily patterns using sleeping trees, their fast gut passage rates and their long feeding bouts. These factors lead to the concentration of a high number of seeds from few maternal sources beneath each sleeping tree, leading to a high rate of genetic relatedness between these seeds.</p>
<p>So we see several factors can lead to a low maternal diversity of accumulated seeds: long feeding bouts, fast passage rate, the use of few fruiting trees before resting.</p>
<p>It’s interesting to note how we can improve our understanding of disperser behavior by analyzing plant spatial genetics. In my opinion, spatial genetics should not be used alone to understand disperser behavior, since there are confounding factors, such as pollination, other seed dispersers, and sometimes even asexual reproduction; but it is a great tool to support inferences we make from animal behavior analysis, concerning the effects of frugivores on plant populations.</p>
<p>Photo: Spider Monkey by Fábio Manfredini – Flickr. Barro Colorado Island from the Smithsonian Institute Webpage si.edu Oenocarpus Batua from Palmpedia.net</p>
<p><a href="images/April7-1.jpg"><img src="images/April7-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April7-2.png"><img src="images/April7-2.png" style="width:30%; margin-right: 20px"></a> <a href="images/April7-3.jpg"><img src="images/April7-3.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 8, 2019</p>
<p>The same Palm tree we saw on the last post can have weaker spatial genetic structure within its seed accumulations when dispersed by male long-wattled umbrella birds (Cephalopterus penduliger) (Karubian 2012). Last month we saw a post on this bird as seed disperser with sex-related individual differences. We will see in this post the effect of its sex-related behavior on the spatial genetic structure within and outside the leks.</p>
<p>Leks are mating related territories, to which male birds always come back after feeding and where most seeds are regurgitated. They spend 80% of their time within these leks and deposit 50% of the seeds. Males constantly come and go to their leks, visiting several fruit trees throughout the day. As a result, the authors found a high number of maternal sources within leks (27 vs 5), and a higher proportion of immigrant coming from outside the study area (44%), increasing genetic diversity and reducing spatial genetic structure within leks.</p>
<p>Other factors that can influence the number of maternal sources within seed accumulations, include the spatial distribution of resources, body size of the dispersers, its social structure and feeding competition.</p>
<p>The number of maternal sources within accumulations of seeds, and their place of origin, can be determined with genetic markers: through parentage analysis of seedlings or through a direct identification of mothers from seed coats.</p>
<p>Spatial genetic structure at the seed stage, and the seedling stage, doesn’t necessarily remain until the reproductive stage. Post dispersal processes will ultimately determine whether spatial genetic structure created through seed dispersal remains over generations.</p>
<p>Links: <a href="https://onlinelibrary.wiley.com/…/…/j.1744-7429.2012.00859.x">Karubian et al. 2012. Mating Behavior Drives Seed Dispersal by the Long-wattled Umbrellabird Cephalopterus penduliger</a> For more on methods for analyzing seed dispersal distance <a href="https://www.biorxiv.org/content/10.1101/528448v1">here</a></p>
<p>Photo:Lars Petersson</p>
<p><a href="images/April8.jpg"><img src="images/April8.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 10, 2019</p>
<p>This time we will discuss an example of how changes in habitat conformation can lead to changes in frugivore behavior and plant spatial genetic structure.</p>
<p>Fontúrbel et al (2018) studied spatial genetic structure in marsupial-dispersed mistletoes in Valdivia (Chile) and compared native habitats versus transformed habitats.</p>
<p>Transformed habitats are habitats where local vegetation has been changed, but not necessarily discontinued as in fragmented habitats, e.g. plantations. This transformation can lead to a series of environmental modifications which cascades into frugivore behavior and plant spatial genetic structure, as was seen by Fontúrbel et al. (2018)</p>
<p>In the study, transformed habitats had a simpler vegetation and a consequent stronger sunlight exposure, which led to the proliferation of many shadow-intolerant species with larger flower displays and fruit crop sizes. This increased fruit availability, changed the dispersers visitation rates, increasing the time spent feeding at the same location 3x in comparison to native habitats. Mistletoes within these habitats had stronger spatial genetic structure and lower genetic diversity than mistletoe populations in native habitats.</p>
<p>The increased time feeding and foraging at the same location increases the number of fruits consumed from each plant individual and reduces the distance between seeds dispersed of this individual, leading to increased genetic relatedness between closely distributed plants. Together with short pollination distances, this likely results in lower genetic diversity within the population, with long term evolutionary implications.</p>
<p>link to paper: <a href="https://www.sciencedirect.com/…/artic…/pii/S0048969718340063" class="uri">https://www.sciencedirect.com/…/artic…/pii/S0048969718340063</a></p>
<p>disperser: “Monito del monte” Dromiciops gliroides Photo by Cristian Benaprés (<a href="https://www.flickr.com/photos/cbenapres/" class="uri">https://www.flickr.com/photos/cbenapres/</a>)</p>
<p>plant: Mistletoe Tristerix Corymbosus Photo by Amico and Nickrent (2009)</p>
<p><a href="images/April10-1.jpg"><img src="images/April10-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April10-2.jpg"><img src="images/April10-2.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 13, 2019</p>
<p>The second method mentioned in the first post, the overlay method, helps us understand genetic discontinuities between populations in relationship to the landscape. In this post we give an example on how this can be used to understand seed dispersal.</p>
<p><a href="https://europepmc.org/article/pmc/pmc2924833">He et al (2010)</a> analyzed the genetic discontinuities in Banksia hookeriana (Proteaceae). Banksia is a fire-killed shrub restricted to isolated sand dunes in south-west Australia generally dispersed by wind for short distances (mostly 10 m).</p>
<p>They analyzed the spatial genetics of individuals from 40 isolated dunes using geneland. The results gave 17 genetic groups within these 40 dunes: 13 genetic groups were a mix of different individuals from dunes, and 4 were groups contianing individuals from two adjacent dunes.</p>
<p>The landscape genetic structure did not match the landscape structure</p>
<p>To understand the genetic connectivity between these dunes, they used Population likelihood assignment tests to infer the seed source population of each individual. They found 5.5% was assigned to a population from a dune outside of their own.</p>
<p>Such strong genetic connectivity, despite the geographical isolation of the metapopulations and short distance seed dispersal, could be explained by wet periods allowing the growth of individuals in areas in-between dune. Since fires happen around every 13 years, time would be enough to maintain inter-generational genetic connectivity.</p>
<p>However, some individuals were assigned up to 2.6 km from the parent population. These long distance assignments could be explained by the presence of cockatoos in the area.</p>
<p>Flocks of Carnaby’s black cockatoos (Calyptorhynchus latirostris) seek out banksias after fires to feed on the seeds that are made more readily accessible by the fire-triggered rupture of the protective woody follicles and removal of camouflaging foliage. Cockatoos often remove the cones and fly away to perch sites elsewhere, sometimes dropping the cones in flight or otherwise at the sites of seed consumption.</p>
<p>This example shows how landscape genetics can also reveal the importance of alternative seed dispersal systems that might seem irrelevant to the naked eye.</p>
<p><a href="images/April13-1.jpg"><img src="images/April13-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April13-2.jpg"><img src="images/April13-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April13-3.jpg"><img src="images/April13-3.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April13-4.jpg"><img src="images/April13-4.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 14, 2019</p>
<p>When it comes to seed dispersal, one of the most important things plant genetics can tell us, is that things are not always what they seem. A good method to understand if what we see is true is the third method of our post on landscape genetics: Parentage analysis</p>
<p>Parentage analysis uses a series of microsatellites or other genetic markers to understand the location of an offspring’s source. This is easy in dioecious plants where we know which parent is the mother, and a bit more ambiguous in monoecious plants where an adult could be either pollen or seed source. In this case, chloroplastic markers are ideal.</p>
<p>A great paper showing the use of parentage analysis unraveling an alternative reality for what we think we are seeing is <a href="https://royalsocietypublishing.org/…/full/10…/rspb.2008.1793">Sezen, Chazdon &amp; Holsinger (2009)</a> Proximity is not a proxy for parentage in an animal-dispersed Neotropical canopy palm.</p>
<p>Toucans are the dominant seed dispersers of Iriartea deltoidea in the study area. Toucans swallow entire fruits and regurgitate intact seeds, they fly long distances but have an average gut retention time of 30 min.</p>
<p>It is usually reasonable to believe that the seedlings growing aggregated below fruiting trees belong to the fruiting trees themselves.</p>
<p>However, when the authors examined the genetic composition of seedlings surrounding reproductive trees, only a few seedlings belonged to the reproductive tree of the area where they were growing. For example, in one sampling plot, 64 seedlings had 27 genetic parents, from which only one seedling belonged to one of the eight reproductive trees found in the sampling plot.</p>
<p>Although seedling patches growing beneath fruiting trees had multiple seed sources, 45–88% of the seedlings were either full- or half-siblings. This indicates dispersers may be repeatedly carrying multiple batches of seeds from the same source trees to the same deposition sites.</p>
<p>Between 160 and 240 reproductive trees occur within the home range of a single toucan (&lt;86ha). These trees have asynchronous fruiting throughout the year, and are the predominant fruiting tree in the sampling area (secondary forest), explaining the high genetic relationship between seedlings dispersed together</p>
<p>The results of this study shows:</p>
<ol style="list-style-type: decimal">
<li><p>When considering seedlings growing beneath fruiting sources as undispersed seeds, the numbers might be over-estimated. Important for modelling methods, especially in vertebrate-dispersed trees. In these cases parentage analysis is necessary to distinguish dispersal limitation.</p></li>
<li><p>Successful dispersal away from parental trees does not necessarily enable escape from high densities of conspecific seedlings or adults, as predicted by the Janzen–Connell hypothesis.</p></li>
<li><p>Spatial distribution of Iriartea deltoidea seedlings is linked to previously described spatial genetic structure; but not because of seed dispersal limitation but because of seed dispersal patterns by frugivores. Despite the long-distance dispersal mechanism usually associated with random distribution of seeds.</p></li>
</ol>
<p>This study is an example of how parentage analysis of plants can help us further understand frugivore behavior and their seed dispersal patterns.</p>
<p>Souce Link: Photo source: Tony Dawe, Andy Morffew, Christian Defferrard and Trinidad Bob.</p>
<p><a href="images/April14-1.jpg"><img src="images/April14-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April14-2.jpg"><img src="images/April14-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April14-3.jpg"><img src="images/April14-3.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April14-4.jpg"><img src="images/April14-4.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 17, 2019</p>
<p>Remember how we said it gets ambiguous using parentage analysis when adults can be both seed and pollen source? (Monoecious). Well, <a href="https://academic.oup.com/jhered/article/103/2/250/892302">Smouse, Sork, Scofield and Grivet (2012)</a> propose a method to make this important analysis more reliable in such cases.</p>
<p>They combine the genetic analysis of seedlings and seeds (pericarp) using nuclear microsatellite markers.</p>
<p>In many cases seedlings usually keep the seed attached for a determined amount of time. The seed coat is maternal tissue, so genotyping it can lead to a direct genetic match with the mother. However, this tissue on seeds attached to seedlings is usually degraded and genotyping it doesn’t yield good results. The condition gets even worse with reduced soil moisture and seeds with longer periods of exposure.</p>
<p>Nonetheless, this decayed tissue and the few genetic results that it may yield, can be used to improve maternal assignment of seedlings.</p>
<p>Smouse et al tested the method on Quercus lobate at the Figueroa Creek valley of Sedgwick Reserve, California, and found that, with every seed coat locus added, the maternal assignation of seedlings increased exponential. With only 3-4 seed coat loci added, the maternal assignation of seedlings reached almost 100% of unambiguity.</p>
<p>They tested seedlings fallen under canopies (gravity), and in open habitat, potentially dispersed by western scrub jays (Aphelocoma californica), western gray squirrels (Sciurus griseus) and small rodents, such as the white-footed mouse (Peromyscus fasciculata).</p>
<p>From the seedlings under canopies (theoretically obviously attributed to the canopy), using only seedling genetic analysis, they got maternal assignment through parentage analysis only for 55.7%. Adding the pericarp analysis to the equation, they increased the maternal assignation to 94%.</p>
<p>Side note: in this study too, parentage analysis revealed not all seedlings found under the canopy belonged to the canopy, “probably due to some secondary animal-vectored movement of acorns between neighboring canopy patches”.</p>
<p>For seedlings in open habitats, where the seed tissue was decayed the most, they went from 50% of maternal assignation using only seedling genetic analysis, to 91% of maternal assignation including pericarp analysis.</p>
<p>So basically, when sampling seedlings, one should take the whole seedling out, and at the base of the root system, where the seed still lays, detach the seed coat and genotype that too, using the same genetic markers and analytical protocol used for the seedlings. Then after the parentage analysis has been ran for all the seedlings, following their methodological description, we would use the pericarp genotype to assign a maternal source to the seedlings with ambiguous maternal assignment by discarding maternal sources assigned by the parentage analysis that do not perfectly match the pericarp genotype. If the list is not reduced to 2 parental candidates, we can allocate maternity based on relative likelihood values.</p>
<p>Extended use for this method: - Use of unilaterally inherited markers (e.g. chloroplast markers) instead of pericarp tissue</p>
<ul>
<li>Determine pollen source, by using the combination to discard maternal assignments and infer paternal assignments.</li>
</ul>
<p>Link paper: Photo source: Alice cummings, Eric Hunt, Oranjepiet, Mikeallred and Don Loarie (Flickr)</p>
<p><a href="images/April17-1.jpg"><img src="images/April17-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April17-2.jpg"><img src="images/April17-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April17-3.jpg"><img src="images/April17-3.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April17-4.jpg"><img src="images/April17-4.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April17-5.jpg"><img src="images/April17-5.jpg" style="width:30%; margin-right: 20px"></a> —-</p>
<p>April 19, 2019</p>
<p>Parentage analysis can be used to assess seed dispersal in fragmented environments. <a href="https://nph.onlinelibrary.wiley.com/…/full/10.1111/nph.14427">Ismail et al. (2017)</a> used it to understand whether effective seed dispersal of white cedars (Dysoxylum malabaricum) by a large frugivore (Malabar grey hornbill: Ocyceros griseus) is long enough to connect small remnant forest patches within an agricultural matrix. These small forest patches are managed and conserved by local communities for worshiping and are called sacred groves.</p>
<p>During their exhaustive sampling of 216 km2 they found no White Cedar trees growing outside of these sacred groves. In fact, White cedars are listed by the IUCN as endangered given its limited distribution range (only present along the coast of South Western India).</p>
<p>Ismail and colleagues analyzed microsatellite markers, assigned parents using CERVUS and calculated the Euclidean distance between mothers, fathers and seedlings. They assigned the closest source as seed source by rejecting the hypothesis that hornbills disperse seeds from the maternal tree directly to the pollen source, through a regression analysis.</p>
<p>The Malabar grey hornbill is known as a relatively strong flier crossing over varied matrices between forest fragments. However they found 99.7% of seed dispersal cases were within-patches, from which 95% within 200m.</p>
<p>The results suggest either feeding and regurgitating by hornbills happens only within the forest patches (Regurgitation times are ca. 10 minutes for white cedar seeds), or that the seeds regurgitated outside forest patches are suppressed due to regular weeding of surrounding plantations.</p>
<p>Nonetheless, the study shows immigration of seeds in patches to be 0.3%, and patches have an average distance of 880m between each other, ranging between 400m and 24.5km, showing that seed dispersal by hornbills might be far shorter than previously predicted through observations.</p>
<p>This difference with previous observations might be because previous studies were done during the breeding season. During the breeding season, male hornbills forage for long distances to feed the females in the nests. Instead, fruiting season of white cedars happens after the juvenile hornbills learn to fly (i.e. fledging). Fledging in other birds has been seen to restrict foraging patterns to dense vegetation. In this case, hornbills would remain within the dense vegetation of the sacred groves, limiting seed dispersal within patches.</p>
<p>Side note: Differences in seed dispersal distances in relationship to behavior changes throughout the year are an interesting point for future research.</p>
<p>Through parentage analysis, the study demonstrates hornbills are not effectively dispersing between patches and this has great implications in terms of conservation for the endangered white cedar.</p>
<p>The lack of recolonization might be further hindered by reduced forest patch attractiveness to hornbills in the absence of fruiting white cedar trees.</p>
<p>White cedars live between 35 to 45 years, so the estimated rare events of inter-patch seed dispersal might contribute to the persistence of the white cedar population. However, 0.3% of inter patch dispersal is likely insufficient to maintain resilient populations without management intervention.</p>
<p>This study is a perfect example of how plant genetics can help determine the effective seed dispersal by a frugivore, which is ultimately what is needed for defining conservation measures.</p>
<p>Photo sources: Aravind Venkatramar, Pierre Grard, IUCN, Mike Charkow, Sasch Ismail.</p>
<p><a href="images/April19-1.jpg"><img src="images/April19-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April19-2.jpg"><img src="images/April19-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April19-3.jpg"><img src="images/April19-3.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April19-4.jpg"><img src="images/April19-4.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April19-5.jpg"><img src="images/April19-5.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April19-6.jpg"><img src="images/April19-6.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April19-7.jpg"><img src="images/April19-7.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 21, 2019</p>
<p>Another example of parentage analysis used to understand seed dispersal by frugivores was done by <a href="https://onlinelibrary.wiley.com/…/…/j.1365-294X.2011.05425.x">Choo, Juenger and Simpson (2012)</a> in the Manu National Park, Peru on Attalea phalerata.</p>
<p>Attalea phalerata is monoecious, but the staminate and pistillate inflorescence on individual palms flower asynchronously so the species is considered functionally dioecious. Pollinated by insects and dispersed primarily by capuchins.</p>
<p>Using CERVUS, they found 73% of offspring where mother were identified were dispersed 30m from the fruiting source. Mean seed dispersal distance was 20.5m. From seeds dispersed beneath fruiting sources 58.5% were maternal siblings. The proportion of neighbour siblings decreased over life stages, likely reflecting density-dependent mortality processes.</p>
<p>The seed dispersal kernels they found reflects the feeding behavior of Capuchins. Capuchins cause a great number of intact fruits to fall to the ground beneath the fruiting sources. Most seeds from the fruits they actually feed on are also discarded beneath fruiting palms, on branches adjacent to palm crowns, before leaving to other fruit sources.</p>
<p>Seeds of Attalea phalerata that are not dispersed away from under parent palms suffer high mortality rates (c. 80%) from subsequent predation by bruchid beetles. The study also analyses spatial genetic structure through the different life stages and found that this seed dispersal pattern created spatial genetic structure that persisted into adulthood.</p>
<p>Photo credit: Canon Foder (Flickr), Rich Hoyer, Christian Defferrard</p>
<p><a href="images/April21-1.jpg"><img src="images/April21-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April21-2.jpg"><img src="images/April21-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April21-3.jpg"><img src="images/April21-3.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April21-4.jpg"><img src="images/April21-4.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 23, 2019</p>
<p>We can also use plant genetics to differentiate how each frugivore from the dispersal guild contributes to the seed shadow of a plant.</p>
<p><a href="https://www.pnas.org/content/104/9/3278">Jordano et al (2007)</a> combined direct observations of fruit consumption, sampling of seed rain by setting seed traps and feces, and linked dispersed seeds to source trees by genotyping seed coats and fruiting trees of Prunus mahaleb.</p>
<p>They identified two components on the seed dispersal kernel. A short-distance component lead by small-sized birds, and a long-distance component lead by medium-sized birds and carnivorous mammals.</p>
<p>In detail, they found: 1) small-sized birds (&lt;110 g) were by far the major seed dispersers of seeds moved &lt;250 m, but rarely dispersed over 100m, while medium-sized birds mostly dispersed over 100m. 2) Larger frugivores (110–500 g) were the major dispersers of seeds moved between 250–990 m. From which carnivorous mammals dispersed seeds between 0 m (i.e., under the canopy of the source tree) to 990 m, with a peak at 650–700 m.</p>
<p>In terms of dispersal distance their most important discovery was that seed immigration from other populations of Prunus mahaleb was mainly through carnivorous mammals (V. vulpes, M. foina, M. meles). Showing critical long-distance dispersal events might rely on a small subset of large species with lower fruit removal rates.</p>
<p>In addition, they found the frequency of seed deposition in different microhabitats differed among frugivores: 1) small-sized birds dispersed seeds mainly beneath the canopies of P. mahaleb and other fleshy-fruited trees or shrubs, where seeds are more likely to establish. 2) Medium-sized birds dispersed seeds mainly to open areas (C. corone) and beneath pine trees (T. viscivorus). 3) Mammals deposited seeds preferentially in open sites (rocky soils and open ground with little woody vegetation or grass cover)</p>
<p>So percentage of immigration of seeds differed between microhabitats following the differential contribution of frugivores to each microhabitat, likely influencing their survival success.</p>
<p>By using plant genetics, this study shows that both components of the seed dispersal kernel have a strong influence on different sides of the seed dispersal curve, therefore, neglecting either component of the seed dispersal kernel based on their removal rates would bias the seed dispersal curve estimated.</p>
<p>Photo credits: K. Zyskowski &amp; Y. Bereshpolova, H vargas, Andy Manning and Vesa Lihavainen.</p>
<p><a href="images/April23-1.jpg"><img src="images/April23-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April23-2.jpg"><img src="images/April23-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April23-3.jpg"><img src="images/April23-3.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April23-4.jpg"><img src="images/April23-4.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 25, 2019</p>
<p>Today we will talk about the future of plant genetics. A recent review suggests microsatellites are an analysis of the past <a href="https://www.ncbi.nlm.nih.gov/pubmed/30575167">(Flanagan &amp; Jones, 2018)</a>, that SNPs are the new microsatellites.</p>
<p>What are microsatellites and what are SNPs?</p>
<p>Microsatellites are short repeat sequences, part of the DNA composed of a sequence with repeated bases. These repeats can be two bases that repeat (ACACACAC), or three, or four (ACGTACGTACGTACGT) and are randomly located throughout the genome. These repeat sequences mutate easily throughout generations, therefore they are a source of genetic polymorphisms within populations.</p>
<p>SNPs instead (pronounced SNIPS) are single nucleotide polymorphisms, so throughout the genome there are these replacements of one single base. For example, instead of ACGT you’d have in some individuals ATGT. These are also a source of polymorphisms within the population, and as such can be used to determine relatedness between individuals. Flanagan &amp; Jones suggest these are the future. Because:</p>
<p>Downsides of Microsatellites: - Initially developed for fishes, and have actually little polymorphism rate in many species. - Require a large initial investment to identify loci, designing primers and optimizing PCR (All ailments I went through during my PhD) - Scoring microsatellites can be ambiguous - Significant investment in terms of labor and financial resources for genotyping</p>
<p>Advantages of SNPs: - Higher polymorphism - Easier automation and scoring - Low mutation rate, so more stability throughout generations</p>
<p>Flanagan and Jones review all the papers that have used SNPs for parentage analysis, many of such papers comparing the use of SNPs vs. Microsatellites and showing SNPs are entirely appropriate for such analysis.</p>
<p>The authors don’t specifically speak about plant parentage analysis but one of the papers they review and many others on the general literature show SNP analyses work in plants too. In fact, plant breeders have been using them for a while apparently.</p>
<p>Once you use SNPs for parentage analysis, since the basis behind plant spatial genetics is scoring relatedness between individuals, you should also be able to use it for intra- and inter-population spatial genetic structure as well, but nobody has done this yet.</p>
<p>The article describes in detail all the different ways of determining parentage analysis and their pitfalls, how to choose a marker, the software and so on, so it is definitely a good read if interested in using such method yourself.</p>
<p><a href="images/April25-1.jpg"><img src="images/April25-1.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April25-2.jpg"><img src="images/April25-2.jpg" style="width:30%; margin-right: 20px"></a> <a href="images/April25-3.jpg"><img src="images/April25-3.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 27, 2019</p>
<p>Speaking about methods, today we will talk about software that will come in useful if you want to use plant genetics to support your projects on seed dispersal.</p>
<p>There are several software available for each method, here are the ones I’ve used while dealing with plant spatial genetics with a link to the scientific article describing them:</p>
<p>1: GenAlEx – Different software use different input files, use GenAlEx to convert your genotype file into the input file for the software of your choice. However, this software can also be used to match genotypes from seed coats to mothers, calculate genetic diversity and allelic variance, estimate genetic differentiation between populations, do mantel correlations, calculate pairwise genetic and geographic distance and much more. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463245/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463245/</a></p>
<p>2: SpaGeDi – If you want to do spatial genetic structure analysis, this is the one. Very simple MS-DOS, you get your converted input file from GenAlEx in the text form add to it whether you want defined distance classes or just a number of distance classes with equal number of pairs inside and save it. Drag the .txt file in the MS-DOS window, choose which coefficient to use, whether you want to jackknife loci for an error rate, how many permutations for determining the confidence intervals and the format of your output. Seconds later you get an output file which you read with excel and can use to calculate the Sp value, and build the spatial autocorrelogram. <a href="https://www2.ulb.ac.be/…/la…/fichiers/manual_SPAGeDi_1-1.pdf" class="uri">https://www2.ulb.ac.be/…/la…/fichiers/manual_SPAGeDi_1-1.pdf</a></p>
<p>3: CERVUS – Use this for parentage analysis you have to use two files one for potential offspring and one for potential parents. You will have to define the percentage of sampled parents, the genotyping error rate and which likelihood method you want. You will get all the potential parents with their own likelihood estimate. If you want to know parent pairs, you select the parents based on the TRIO LOD score, not each parent’s likelihood score, I had to email the developer to figure this one out. <a href="https://www.ncbi.nlm.nih.gov/pubmed/17305863" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/17305863</a></p>
<p>4: COLONY – This one you’d use to determine sibling relationships, also just parents but CERVUS is faster for parents. COLONY instead can be used, for example, to determine the number and location of maternal sources for seeds collected from seed traps/under sleeping trees, or for estimating the extent of seed shadows by finding maternal siblings. COLONY is very very straight forward to use with a user-friendly interface, it’s just very slow if you select the most statistically powerful options, so start with the fast analysis if you want to just try it out. <a href="https://www.ncbi.nlm.nih.gov/pubmed/21565056" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/21565056</a></p>
<p>5: STRUCTURE - Use this to assess patterns of genetic structure, especially at large scales, for example, when comparing populations. It detects differences in allele frequencies and puts individuals into a number of K clusters, then you use this to determine whether the populations are composed of different clusters or not. It has a very graphical output which you can then plot on a map using GIS. I loved it when I used it, but I didn’t include the results in my thesis. I used it coupled with CLUMPAK to select which was the most feasible number of K clusters from the several options given by the output of STRUCTURE. Here is a nice paper describing the use of STRUCTURE and the coupled software. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3665925/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3665925/</a></p>
<p>This are the ones I’ve personally used, of course there are many others and the choice should be specific to your project’s needs.</p>
<p><a href="images/April27.jpg"><img src="images/April27.jpg" style="width:30%; margin-right: 20px"></a></p>
<hr />
<p>April 29, 2019</p>
<p>This is our last post of the month on plant genetics, we’ll take it to make a summary of what we’ve seen this month and discuss why studying plant genetics is important.</p>
<p>This month, we introduced landscape genetics, the different methods within and how these can be used to understand further seed dispersal by frugivores.</p>
<p>We talked about genetic vs geographic distance correlations (i.e. spatial genetic structure) which can be analyzed between populations, within populations, and even within seed accumulations, to understand gene flow restriction, and seed dispersal patterns.</p>
<p>We saw how through these methods it’s been uncovered that not every accumulation beneath each fruiting tree is composed of undispersed trees, and not every accumulation away from fruiting trees is composed of unrelated seeds. This will strictly depend on frugivore behavior before and after feeding.</p>
<p>Then we talked about Bayesian models overlaid to maps, clustering genetic information into groups, and determining similarity between populations, to understand seed migration and long distance seed dispersal rates.</p>
<p>Finally, we talked about parentage analysis, how it can be done with seed coats to reveal maternal sources, or with seedlings to reveal maternal and parental sources. This can be used to determine seed dispersal distance, seed shadow extent of a determined individual, number of maternal sources within seed accumulations and even to compare effective seed dispersal distance vs basic seed dispersal distance.</p>
<p>We also saw what the future holds for plant genetics, that we should all start thinking about SNPs instead of microsatellites or AFLPs (AFLPs were already old school by the way), and we finished with a short introduction to the most used software for plant spatial genetics.</p>
<p>We didn’t talk about why understanding gene flow is important. So, if you are still skeptical plant genetics is worth your time, the ultimate reason is that you can understand the long-term consequences of your seed dispersal system.</p>
<p>If your seed dispersal system, perhaps because it is fragmented or because it has been defaunated, is restricting gene flow (i.e. you see strong spatial genetic structure, no seed migration, short dispersal distance) this can lead to inbreeding within populations, losing genetic diversity, reducing the population’s fitness, and increasing the extinction risk of the population in case of adverse events (i.e. climate change). An important link between conservation, frugivores and seed dispersal.</p>
<p>Gene flow has also been considered an important evolutionary force, but for that regard, I invite you to read <a href="https://bsapubs.onlinelibrary.wiley.com/…/10.37…/ajb.1400024" class="uri">https://bsapubs.onlinelibrary.wiley.com/…/10.37…/ajb.1400024</a></p>
<p><a href="images/April29.jpg"><img src="images/April29.jpg" style="width:30%; margin-right: 20px"></a></p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
